packages/unmock-node/src/types.ts

```diff
@@ -3,5 +3,4 @@ export {
   ISerializedRequest as Request,
   ISerializedRequest as UnmockRequest,
   ISerializedResponse as UnmockResponse,
-  States,
```

Comment:
Maybe expose  `Services` then?

Reply:
Hm, sure. I don't see pros or cons to exposing it (do we expect users to manually type it when it's returned from `unmock.services` as `ServiceStoreType`?)

Reply:
I think the types of all exposed values should be exposed, so users don't need to use `any` if doing 
```
let services: any;
beforeAll(() => {
   services = unmock.on().services;
}
```

Reply:
The bravest hackers would of course do something like `let services: typeof unmock.services`.

Reply:
Good point.

Also, since `services` is a dictionary, one can safely do `beforeEach(() => unmock.on());` now (I guess it tries to access `unmock.services.then`, but that's just a no-op).